tests/lib/apt.sh

```diff
@@ -2,6 +2,9 @@
 
 . $TESTSLIB/pkgdb.sh
 
+curr_history=/var/log/apt/history.log
```

Comment:
tests/lib/apt.sh is being dropped in https://github.com/snapcore/snapd/pull/3406 and merged into tests/lib/pkgdb.sh

Each new part we're going to add related to packaging needs to go into tests/lib/pkgdb.sh and needs to have proper variants for each of the support distributions.